test/gpu/jit.cpp

```diff
@@ -383,13 +383,10 @@ TEST_CASE(compile_math)
     auto vec_sizes = {2, 4, 6};
     for(auto&& t : migraphx::shape::types())
     {
-        if(contains({migraphx::shape::bool_type,
-                     migraphx::shape::tuple_type,
-                     migraphx::shape::bf16_type},
-                    t))
+        if(contains({migraphx::shape::bool_type, migraphx::shape::tuple_type}, t))
             continue;
         auto name = migraphx::shape::cpp_type(t);
-        if(t == migraphx::shape::half_type)
+        if((t == migraphx::shape::half_type) or (t == migraphx::shape::bf16_type))
```

Comment:
You could write this as `if(contains({migraphx::shape::half_type, migraphx::shape::bf16_type}, t))`